app/components/FilterableComponents/FilterableTablePagination.tsx

```diff
@@ -1,42 +1,65 @@
 import React from 'react';
 import {useRouter} from 'next/router';
-import {Pagination} from 'react-bootstrap';
+import {Pagination, Dropdown} from 'react-bootstrap';
 
 interface Props {
   totalCount: number;
-  maxResultsPerPage: number;
 }
 
-export const FilterableTablePaginationComponent: React.FunctionComponent<Props> = (
+export const DEFAULT_MAX_RESULTS = 20;
+
+const FilterableTablePaginationComponent: React.FunctionComponent<Props> = (
   props
 ) => {
-  const {totalCount, maxResultsPerPage} = props;
+  const {totalCount} = props;
   const router = useRouter();
-  const maxPages = Math.ceil(totalCount / maxResultsPerPage);
 
-  // Set the activePage by query string value. If no value exists, the activePage is the first page
-  let activePage = 1;
-  if (router?.query?.pageVars)
-    activePage =
-      JSON.parse(router?.query?.pageVars?.toString())?.activePage || 1;
+  const getPageData = () => {
+    try {
```

Comment:
should we use the `safeParse()` function here?

Reply:
this is functionally the same with the getPageData() using a try/catch block, but if we split safeParse() into a `/lib` then it could definitely be reused here instead of re-writing essentially the same functionality in this component